iceoryx_meta/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
-# Copyright (c) 2019, 2020 by Robert Bosch GmbH, Apex.AI Inc. All rights reserved.
+# Copyright (c) 2019, 2020 by Robert Bosch GmbH. All rights reserved.
```

Comment:
Very good! The only thing missing is the SPDX license statement at the end of the fileheader. I know you will bite into your keyboard and wish me all the best, but to cite a famous captain:
> It is possible to commit no mistakes and still lose. That is not a weakness—that is life.
—”Peak Performance” (TNG S02 Ep. 21)

example:
```cpp
    // See the License for the specific language governing permissions and
    // limitations under the License.
    //
    // SPDX-License-Identifier: Apache-2.0
````

Reply:
@dkroenke Aye Aye Capt'n. - I will make it so!

And this citation actually reliefs some of the pain I am feeling right now.